less/components/content-container.less

```diff
@@ -4,14 +4,12 @@
     position: relative;
     overflow: auto;
     z-index: 1;
-    padding: 85px 0 70px 0;
```

Comment:
This component is being reused (and that's it's purpose)

This padding thought was to solve design issues in *only* the web fellows page. So you can see me adding this page's changes to the `less/pages/open-web-fellows/shared.less` file